app/components/dex-create.jsx

```diff
@@ -84,15 +84,15 @@ export class DexCreate extends Component {
   }
 
   render () {
-    const { games, gamesById, isOpen, session } = this.props;
+    const { games, gamesById, isOpen, session, nightMode } = this.props;
```

Comment:
looks like the modals get added outside of root, so i had to maually add this to both `dex-create` and `dex-edit`. open to doing something else if you see a better option!